linkerd/namer/k8s/src/main/scala/io/l5d/k8s.scala

```diff
@@ -73,6 +73,7 @@ case class k8sConfig(
     // namer path -- should we just support a `label`?
     val path = prefix.show
     val service = client
+      .withParams(params)
```

Comment:
huh, transform only operates on the stack, so withParams isn't an option here. gonna go with a `.copy()` instead, I think